### PR TITLE
NO-ISSUE: Gather logs using sos report on nodes

### PR DESCRIPTION
**Pull request by adriengentil**
Use sos report along with toolbox to gather the logs from cluster
nodes

### CODE REVIEW

File: src/assisted_test_infra/download_logs/resources/man_sosreport.sh [hunk 1]
```diff
@@ -1,135 +1,15 @@
...
 
-echo -e "Gathering system information..."
-hostname &> hostname
-cp -a /etc/redhat-release  ./etc/ 2>> error_log
-uptime &> uptime
+sudo toolbox sos report --batch --tmp-dir "${SOS_TMPDIR}" --compression-type xz --all-logs \
+                   -o processor,memory,container_log,filesys,logs,crio,podman,openshift,openshift_ovn,networking,networkmanager,rhcos \
+                   -k crio.all=on -k crio.logs=on \
+                   -k podman.all=on -k podman.logs=on \
+                   -k openshift.with-api=on
```
Comment (osherdp) [inline src/assisted_test_infra/download_logs/resources/man_sosreport.sh:12]: I saw there's an option to set a global timeout for all plugins
Maybe we should do so to make sure we always have as much data as we can?

**Comment by adriengentil** [comment]
/close
in favour of https://github.com/openshift/assisted-test-infra/pull/1943

**Pull request closed without merging**